src/main/resources/webui/css/auditing.css

```diff
@@ -0,0 +1,74 @@
+#auditing-div {
+    display: flex;
+    height: 100%;
+}
+
+#auditing-output-div {
```

Comment:
try not to style `id`s but `class`es. It's not good practice. you can leave it like that as it doesn't affect the app.